### ISSUE TITLE
The `append_body` input not released

### ISSUE THREAD

**Issue by hendrikmaus** [softprops/action-gh-release#216]
I noticed that the `append_body` input (https://github.com/softprops/action-gh-release/pull/199) is not yet released. When are you planning to publish it?

**Comment by abcfy2** [comment]
Yes. So you have to use `softprops/action-gh-release@master` instead of `softprops/action-gh-release@v1`

**Comment by hendrikmaus** [comment]
As a workaround, yes, but not on the long run.

**Comment by wolf-michl** [comment]
Hi!
When do you plan to release this feature? For obvious reasons I can't just use `softprops/action-gh-release@master` - I need to pin it to a proper version. I would need this new `append_body` feature for an issue of mine.
Thanks in advance for letting me know!

**Comment by MathiasVandePol** [comment]
@wolf-michl You can pin it to `softprops/action-gh-release@cd28b0f5ee8571b76cfdaa62a30d51d752317477`

**Comment by chenrui333** [comment]
Cannot reproduce this issue with the latest `v2.5.2` release. If you're still seeing a different failure on current releases, please open a new issue with a fresh repro or sanitized workflow snippet so we can investigate it separately.
